### ISSUE TITLE
Needs more explanation about usage/synchronisation

### ISSUE THREAD

**Issue by HubKing** [mcxiaoke/ESPDateTime#15]
So.... if I call DateTime.begin() once in setup() just like the top page's example, does it periodically re-synchronise the time, or does it only do that once at begin() and never re-synchronise?

**Comment by mianos** [comment]
No, seems you need to call begin() again. 
```
unsigned long lastInvokeTime = 0;
const unsigned long dayMillis = 24UL * 60 * 60 * 1000; // Milliseconds in a day
...
// in loop()
  if (currentMillis - lastInvokeTime >= dayMillis) {
      DateTime.begin(1000);
      lastInvokeTime = currentMillis;
  }
 ```

**Comment by HubKing** [comment]
How much does an ESP's timer generally deviate in a day?

**Comment by mianos** [comment]
I have seen a second or so over time time.
I don't know but it is just the 'vibe' to me.

**Comment by infomaniac50** [comment]
**TLDR: The Espressif SDK, and by extension this library, periodically synchronizes the time every hour.**

The Espressif SDK uses the lwIP SNTP library internally. Any application that initializes the SNTP service periodically synchronizes the time. The time synchronization period is determined by CONFIG_LWIP_SNTP_UPDATE_DELAY (the default value is one hour).

https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/api-reference/system/system_time.html#sntp-time-synchronization
